swarm/storage/localstore.go

```diff
@@ -194,20 +194,21 @@ func (ls *LocalStore) Close() {
 	ls.DbStore.Close()
 }
 
-// Migrate checks the datastore schema vs the runtime schema, and runs migrations if they don't match
+// Migrate checks the datastore schema vs the runtime schema and runs
+// migrations if they don't match
 func (ls *LocalStore) Migrate() error {
 	actualDbSchema, err := ls.DbStore.GetSchema()
 	if err != nil {
 		log.Error(err.Error())
 		return err
 	}
 
-	log.Debug("running migrations for", "schema", actualDbSchema, "runtime-schema", CurrentDbSchema)
```

Comment:
I actually like this log message, because even if migrations are not run, it tells you about your current schema. I don't think we have another log message which gives the current schema to the user.

So it might make sense to update the `running migrations for` string, but the location should stay as is.

Reply:
This got merged before addressing your comment, @nonsense.
I can piggyback your suggestion to another PR if you wish.